repose-aggregator/core/core-lib/src/main/java/org/openrepose/powerfilter/PowerFilter.java

```diff
@@ -374,6 +378,10 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
                 }
                 requestFilterChain.startFilterChain(mutableHttpRequest, mutableHttpResponse);
             }
+        } catch (IllegalArgumentException iae) {
```

Comment:
I agree see if you can't scope it down, so that it's only catching the exception at the call level. Without looking in depth at the code you may end up catching it, and then wrapping it in a new exception and throwing that to catch where you are currently catching this one